platformio.ini

```diff
@@ -31,7 +31,7 @@ build_flags = -fmax-errors=5
 lib_deps =
   https://github.com/MarlinFirmware/U8glib-HAL/archive/dev.zip
   LiquidCrystal@1.3.4
-  https://github.com/teemuatlut/TMCStepper.git
+  TMCStepper@<1.0.0
```

Comment:
I can see why you may prefer this. However, I really hope in the near future to get `platformio.ini` back to the point where it wasn't required to download and install `git` on Windows, which is another hoop for novice users to jump through.

Reply:
It doesn't. I did try this in a VM before committing the change.

Reply:
To clarify, direct `git` links do need it. But when specifying just the registered library name, PIO can handle the download itself (through whatever) and git is not needed.

Reply:
Right, so I guess there are only certain libraries (those that PlatformIO doesn't know about) that need special treatment.